src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -106,6 +106,8 @@ public function read(array $domains, array $locales): TranslatorBag
                     'locale' => $locale,
                     'domain' => $domain,
                 ];
+
+                $response->getStatusCode(); // make the request synchronous to prevent error 429 from Loco API.
```

Comment:
no need for that: merging the two foreach would achieve the same (and allow removing the $responses array

Reply:
Yes, the two foreach were implemented to make the process async, but as we need to make it sync, let's remove one foreach :+1:

Reply:
I thought about that but didn't wanted to change too much lines.

Gonna change it :+1: